support-frontend/assets/helpers/productCatalog.ts

```diff
@@ -143,16 +139,6 @@ const guardianWeeklyBenefit = {
 	copy: 'Guardian Weekly print magazine delivered to your door every week  ',
 	tooltip: `Guardian Weekly is a beautifully concise magazine featuring a handpicked selection of in-depth articles, global news, long reads, opinion and more. Delivered to you every week, wherever you are in the world.`,
 };
-const newspaperArchiveBenefitUK = {
```

Comment:
the newspaper archive benefits test will be done from the tool

Reply:
Is the plan to move all the benefits into the tool eventually?

Reply:
For now it's just the "3 tier" products. So the print products config remains here, for now...